signwriting_animation/diffusion/core/distribution.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch import nn
+
+class DiagonalGaussianDistribution:
```

Comment:
what makes it diagonal?
Simple sampling model: https://github.com/sign-language-processing/transcription/blob/main/text_to_pose/diffusion/src/model/distribution.py